feeadjuster/feeadjuster.py

```diff
@@ -234,8 +262,10 @@ def feeadjust(plugin: Plugin):
     channels_adjusted = 0
     for peer in plugin.peers:
         for chan in peer["channels"]:
-            if chan["state"] == "CHANNELD_NORMAL":
-                scid = chan["short_channel_id"]
+            if chan["state"] in ["CHANNELD_NORMAL", "CHANNELD_AWAITING_LOCKIN"]:
+                scid = chan.get("short_channel_id")
```

Comment:
`CHANNELD_AWAITING_LOCKIN` is here because of a clightning bug? So this meant to be temporal till the bug gets fixed?

Reply:
Actually this was more like testing. `channel_state_changed` will not trigger for new channels (meaning new_state `CHANNELD_AWAITING_LOCKIN` and old_state `UNKNOWN???`. We need to use the `channel_opened` notification for this, buuuut when a channel is new (not locked in) we don't have a `scid` but only the `funding_txid`.

Ideally we call `setchannelfee` when a new channel it is still in `CHANNELD_AWAITING_LOCKIN`, because then we don't send two updates shortly after one another, one with default fees, and then the modified fees (which can cause problems for the gossip network). But in order to be able to do this we need to wait for `channel_opened` notification for which we only have `funding_txid` and no `scid`. So we need to change more code of this plugin so it deals with `funding_txid` instead of `scid`.

I will convert this to draft until this is fiddled out. But still good to have some discussion about this...